node-util/sbin/oo-init-quota

```diff
@@ -79,11 +79,11 @@ QUOTA_OPTIONS=usrjquota=aquota.user,jqfmt=vfsv0
 function update_fstab() {
     # ORIGIN_MOUNTPOINT=$1
 
-    if ! tr -s ' ' < /etc/fstab | egrep "^[^#].+ $1 ext4 [^ ]+$QUOTA_OPTIONS "; then
+    if ! tr -s '[:blank:]' < /etc/fstab | egrep "^[^#].+\s+$1\s+ext4\s+.*$QUOTA_OPTIONS\s+"; then
```

Comment:
Could we change the last `\s+` to `[[:blank:],]+`? Otherwise, this pattern fails to match if `$QUOTA_OPTIONS` is followed by further options.